CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Add an instructor widget to modify join the discussion mode
 - Prevent join API calls in join mode denied
 - Add a forced join mode which automatically joins students to the discussion
+- Manage live media sharing with picture-in-picture on student side
+
```

Comment:
SHouldn't this be in the unreleased section ?

Reply:
yes, ty